astropy/io/fits/scripts/fitsheader.py

```diff
@@ -42,14 +42,11 @@
 With Astropy installed, please run ``fitsheader --help`` to see the full usage
 documentation.
 """
-from __future__ import (absolute_import, division, print_function,
-                        unicode_literals)
```

Comment:
It applies only to the file in which it is imported, so it's possible to have modules where it is not used. And most of the time it's useless, or it causes more issues than it solves. So here I think it's safe to remove it, and it fixes the uses of fitsheader with an invalid file.